readme.md

```diff
@@ -110,6 +110,7 @@ Colors can be changed by using [`zstyle`](http://zsh.sourceforge.net/Doc/Release
 - `path` (blue) - The current path, for example, `PWD`.
 - `prompt:error` (red) - The `PURE_PROMPT_SYMBOL` when the previous command has *failed*.
 - `prompt:success` (magenta) - The `PURE_PROMPT_SYMBOL` when the previous command has *succeded*.
+- `prompt2:faded` (242) - The color for showing the state of the parser in the continuation prompt (PS2).
```

Comment:
Would be useful to include a use-case for wanting to change this color. (Your virtualenv use-case, for example)

Reply:
> Can you fix the merge conflict and add the color setting to the ASCII art that shows where in the prompt it is?

I’ll fix it when I get home today or tomorrow. Regarding the ASCII art, I’m not sure where / how to display it. Adding the continuation prompt there (new line) could look a bit weird. And it’s also not entirely correct to put it on the same line as virtualenv. Perhaps we could add a note of colors “not pictured” in the graph?

Reply:
Make the color pink, take a screenshot of your prompt with it and describe that it’s the pink part and link to the screenshot?